arrowhead/arrowhead.rb

```diff
@@ -15,20 +14,30 @@ class Arrowhead
     },
   }
 
+
+class Arrowhead
+
+  attr_reader :shape, :region_info
+
   # FIXME: I don't have time to deal with this.
```

Comment:
Haha - I'm not that mean!

Those came with the code, so I probably should have deleted them! Noted!

On Tue, Sep 29, 2015 at 3:52 PM, Benjamin Feng <notifications@github.com>
wrote:

> In arrowhead/arrowhead.rb
> <https://github.com/fengb/level_up_exercises/pull/13#discussion_r40727777>
> :
>
> > @@ -15,20 +14,30 @@ class Arrowhead
> >      },
> >    }
> >
> > +
> > +class Arrowhead
> > +
> > +  attr_reader :shape, :region_info
> > +
> >    # FIXME: I don't have time to deal with this.
>
> Deal with it!
>
> (Probably was fixed... If the comment makes no sense, it's probably stale
> and should be deleted. Stale comments are worse than no comments.)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/fengb/level_up_exercises/pull/13/files#r40727777>.
>